l10n_ec_ote/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Bug Tracker
```

Comment:
@coodec Don't worry, it will work once we merge a module.

https://runbot.odoo-community.org/runbot/repo/github-com-oca-l10n-ecuador-212